plugins/Oculars/src/Oculars.cpp

```diff
@@ -1811,8 +1811,8 @@ void Oculars::paintCCDBounds()
 	double azimuth, elevation;
 	StelUtils::rectToSphe(&azimuth, &elevation, centerPos3d);
 	const auto derotate = Mat4f::rotation(Vec3f(0,0,1), azimuth) *
-						  Mat4f::rotation(Vec3f(0,1,0), -elevation) *
-						  Mat4f::rotation(Vec3f(1,0,0), ccd->chipRotAngle() * (M_PI/180));
+			      Mat4f::rotation(Vec3f(0,1,0), -elevation) *
```

Comment:
Why did you un-align these (and the similar lines below)?

Reply:
Hmm... these lines aligned now for me. Did you used `stellarium-ide.xml` file for QtCreator?

Reply:
I don't use QtCreator with Stellarium. But IIRC, we use size-4 tabs, which then should make my indentation align the factors. Here's how your patch looks for me:

![Screenshot_2023-06-23_21-29-09](https://github.com/Stellarium/stellarium/assets/6376882/6d242f87-1fe8-4ec6-b439-b4ecfa72f837)

Reply:
Same place within QtCreator with our standard settings from `stellarium-ide.xml` file:
<img width="1076" alt="Снимок экрана 2023-06-24 в 00 39 32" src="https://github.com/Stellarium/stellarium/assets/88731/4b76e37b-1da4-4f2d-ab3e-759a2013d029">

Reply:
Wait, we have 8-position tabs?! So wasteful! And BTW, this is not mentioned in the coding style document, only in the XML file.
OK, I'll alter my Vim settings, and you please separate this reformatting from the PR, so that git-blame and git-show showed a clearer picture of what this PR really does.